client/scss/components/forms/_field.scss

```diff
@@ -72,3 +72,16 @@
     padding-inline-start: $text-input-height;
   }
 }
+
+// Display these as inline block so that action icons such as comments can appear as close as possible
+// We remove the padding here to let the chooser itself handle the padding
+.w-field--model_choice_field .w-field__input {
```

Comment:
Here and below – I think we should also stop using field types and use widget types instead, as that’s what affects the UI.